library/globals.inc.php

```diff
@@ -1792,6 +1792,7 @@ function gblTimeZones()
                 '2' => xl('Last name, first name'),
                 '3' => xl('Last name, first name (title)'),
                 '4' => xl('Last name, first name (title: comments)'),
+                '5' => xl('Last name, first name, address (title: comments')
```

Comment:
just need a parenthesis at end of string and then this is ready for codebase:
```
'5' => xl('Last name, first name, address (title: comments)')
```

Reply:
oh, careless!! - thanks.